src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -181,22 +188,25 @@ object SparkCassandraStress {
       }
 
     val wallClockStartTime = System.nanoTime()
-    val time = test.runTrials(sc)
+    val timesAndOps: Seq[(Long,Long)]= test.runTrials(sc)
+    val time = for (x <- timesAndOps) yield {x._1}
+    val totalCompletedOps = for (x <- timesAndOps) yield {x._2}
```

Comment:
The return type of runTrials returns a list of tuples now (runtimes, nops) so we need to massage things a bit to get the info we want.